webextensions/api/webRequest.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤44"
```

Comment:
@queengooborg The earliest I can trace this back to is 58 (e.g. https://github.com/mdn/browser-compat-data/pull/324[)](https://github.com/mdn/browser-compat-data/commit/82b72201e181b8aad847f7904f1c08bda567cb64))

Reply:
Ah, this was set to 44 because of a child feature that's set to 44!